sw/tools/rtp_viewer/rtp_viewer.py

```diff
@@ -72,7 +78,7 @@ def on_key(self, key):
             self.mouse['start'] = None
 
     def on_mouse(self, event, x, y, flags, param):
-        if event == cv2.EVENT_LBUTTONDOWN and self.rotate == 0:
+        if event == cv2.EVENT_LBUTTONDOWN and self.rotate == 0 and False:
```

Comment:
@kirkscheper the 'and False' part was added in IMAV to disable the mouse event, since it has no use. In order to make the VIDEO_ROI message usable I believe we also need to account for stream downsize, opencv scale & rotation and stream port (?to know aircraft and camera). Maybe this is for a different PR though. What do you think?